homeassistant/components/starline/services.yaml

```diff
@@ -8,3 +8,10 @@ set_scan_interval:
     scan_interval:
       description: Update frequency (in seconds).
       example: 180
+set_scan_obd_interval:
+  description: >
+    Set update OBD info frequency.
```

Comment:
Should this be "Set OBD update frequency." "Set OBD info update frequency." or "Set update frequency for OBD info."?